.github/issue_template/question.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Asking a question / requesting help
+about: Ask the community for help or advise
```

Comment:
advise -> advice

Reply:
fixed in https://github.com/fNIRS/snirf/pull/138/commits/6d16f1caa2ed5186c5f5597266c1f08fa3e9997c